Sources/GitBuddyCore/Commands/ReleaseCommand.swift

```diff
@@ -33,6 +33,9 @@ struct ReleaseCommand: ParsableCommand {
     @Option(name: .shortAndLong, help: "The last release tag to use as a base for the changelog creation. Default: previous tag.")
     var lastReleaseTag: String?
 
+    @Option(name: .customLong("changelogToTag"), help: "If set, the date of this tag will be used as the limit for the changelog creation. This variable should be passed when `tagName` is set. Default: latest tag.")
```

Comment:
It actually has!

```swift
let changelogToTag = try changelogToTag.map { try Tag(name: $0) } ?? Tag.latest()
```

The variable is required if a specific `tagName` is set since the latest tag is yet to be created. 

There are basically two scenarios:
- Creating a release after a tag was already created
- Creating a release and with that creating a tag using `tagName`

For the first scenario, it's clear that the changelog should be created using range `[previousTag - lastTag]`. For the second scenario, however, we're creating a release from a certain commit or branch which we can't use as trailing boundary for the changelog creation. Therefore, it's required to help `GitBuddy` by informing which boundary tag to use.

Let me know if that clarifies things!

Reply:
Gotcha! I got confused about the default because the specific option variable itself did not have a default. 😄

Reply:
Ah yeah, I see! I guess the default value is not always provided through the command interface. Especially in this case, since it performs actual git operations at the moment of execution :)